### PR TITLE
paulinaacuna's hometown file

### PR DESCRIPTION
**Pull request by oblea**
Will be adding my recommendations for places to eat in Chicago.

### CODE REVIEW

**Comment by oblea** [comment]
add hometown files for recommendations

Closes #12

File: paulinaacuna-hometown.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Chicago
+
+## Places to Eat
+
+- Gyu Kaku KBBQ
```
Comment (ROBERTJRINGHAM) [inline paulinaacuna-hometown.md:5]: This looks especially promising!

Comment (ROBERTJRINGHAM) [inline paulinaacuna-hometown.md:5]: What is your favorite dish here?

**Review by ROBERTJRINGHAM — CHANGES_REQUESTED** [review]
I'll have to take your word for these.  Been to 'Chicago' 4 times, never made it much outside of airport or Riverwoods facility.  Though *KBBQ* sounds great!

**Review by ROBERTJRINGHAM — APPROVED** [review]

**Pull request merged**